### PR TITLE
Only report Codecov status on pulls

### PR DESCRIPTION
**Pull request by spawnia**
Otherwise, it makes the repository page look really bad.

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+coverage:
+  status:
+    project:
+      default:
+        only_pulls: true
+comment: false
```
Comment (simPod) [inline .codecov.yml:6]: this is IMO useful, provides quick overview on what's changed in a comment

**Review by simPod — COMMENTED** [review]
How does it look bad? Not sure what this change does exactly 🤔

**Comment by spawnia** [comment]
For some reason, codecov keeps failing even on config only changes and shows the ci of the project page to be failing. When I am shopping for packages, an always green pipeline is something I value.

File: .codecov.yml (hunk 1 above)
Comment (spawnia) [reply .codecov.yml:6]: I am fine with keeping them in, sure

**Comment by simPod** [comment]
Got it. I'll try to come up with a fix and meanwhile we can ship this.

**Pull request merged**